pkgs/applications/window-managers/fvwm/default.nix

```diff
@@ -1,28 +1,46 @@
-{ gestures ? false
-, lib, stdenv, fetchurl, pkg-config
-, cairo, fontconfig, freetype, libXft, libXcursor, libXinerama
-, libXpm, libXt, librsvg, libpng, fribidi, perl
-, libstroke ? null
-}:
+{ autoconf, automake, gestures ? false, lib, stdenv, fetchFromGitHub, pkg-config
+, cairo, fontconfig, freetype, libXft, libXcursor, libXinerama, libXpm, libXt
+, librsvg, libpng, fribidi, perl, libstroke ? null, readline, libxslt }:
```

Comment:
```suggestion
, librsvg, libpng, fribidi, perl, libstroke, readline, libxslt }:
```

Reply:
I don't remember why, but I wanted to keep gestures optional.  We probably could just unconditionally require libstroke, but then we'd probably also want to remove the gestures boolean option.

Reply:
>  I wanted to keep gestures optional.

If you set gestures to true and libstroke is behind lib.optional no such thing is required and only shadows bugs.

Reply:
Done.